src/_provider/Simkl/single.ts

```diff
@@ -56,12 +59,40 @@ export class Single extends SingleAbstract {
   }
 
   _setStatus(status) {
-    if (status === 23) status = 1;
+    if (status === definitions.status.Rewatching) {
+      status = definitions.status.Watching;
+    }
     status = helper.translateList(status, parseInt(status.toString()));
-    if (status !== this.animeInfo.status) this.statusUpdate = true;
+    if (status !== this.animeInfo.status) {
+      this.statusUpdate = true;
+    }
     this.animeInfo.status = status;
   }
 
+  _setStartDate(startDate) {
+    // Unsupported
+  }
+
+  _getStartDate() {
+    return null;
+  }
+
+  _setFinishDate(finishDate) {
+    // Unsupported
+  }
+
+  _getFinishDate() {
+    return null;
+  }
+
+  _setRewatchCount(rewatchCount) {
+    // Unsupported
+  }
+
+  _getRewatchCount() {
+    return null;
+  }
+
```

Comment:
Probably better to just throw an error unsupported. As we do not want the code to get to this point

Reply:
Fixed